Tribler/Core/Modules/restapi/multichain_endpoint.py

```diff
@@ -294,10 +295,12 @@ def render_GET(self, request):
             return MultichainNetworkEndpoint.return_error(request, message="focus_node parameter empty")
 
         focus_node = request.args["focus_node"][0]
+        user_node = hexlify(mc_community.my_member.public_key)
         if isinstance(focus_node, basestring):
             if request.args["focus_node"][0] == "self":
                 if mc_community.persistence.dummy_setup:
                     focus_node = "00"
+                    user_node = "00"
                 else:
                     focus_node = hexlify(mc_community.my_member.public_key)
```

Comment:
Maybe change this to ```focus_node = user_node```, as ```user_node``` should still be the hex of ```mc_community.mymember.public_key```, but this might look unclear. Do it at your own judgement